src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/NetworkHelper.kt

```diff
@@ -555,4 +556,16 @@ class NetworkHelper(
             }
         }
     }
+
+    internal fun apiStateParams(): IMap<String, Any>? {
```

Comment:
Nit: return type can be non-optional, e.g,. `IMap<String, Any>`

Reply:
mm yea our ParsingHelper merge functions take in that specific type... not sure what's the best way to cast/not cast t_t

Reply:
will send it for nowwww!